crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -161,6 +160,24 @@ where
 }
 
 impl ConductorConfig {
+    /// The most minimal config, with in-memory networking
+    pub fn empty() -> Self {
```

Comment:
"empty" conductor config is not expressive to me. Something like "minimal", "in_memory"?

Reply:
removed.